tests/foreman/api/test_activationkey.py

```diff
@@ -439,7 +438,6 @@ def test_positive_remove_user():
 
 @pytest.mark.upgrade
 @pytest.mark.run_in_one_thread
-@skip_if_not_set('fake_manifest')
```

Comment:
Where is this check happening now?

Reply:
It doesn't look like the test actually uses fake_manifest, but I can add this back in to make sure we're keeping it consistent.